libs/experimental/langchain_experimental/graph_transformers/diffbot.py

```diff
@@ -146,6 +147,7 @@ def __init__(
         fact_confidence_threshold: float = 0.7,
         include_qualifiers: bool = True,
         include_evidence: bool = True,
+        include_confidence: bool = False,
```

Comment:
This is a breaking change since there's no `*` usage, place at end of list after a `*` perhaps?

Reply:
Fixed